docker-compose.ci.yml

```diff
@@ -13,7 +13,8 @@ services:
       BOT_TOKEN: ${BOT_TOKEN}
       DJANGO_SECRET_KEY: ${DJANGO_SECRET_KEY}
       OPEN_WEATHER_API_KEY: ${OPEN_WEATHER_API_KEY}
-      OPENAI_API_KEY: ${OPENAI_API_KE}
+      OPENAI_API_KEY: ${OPENAI_API_KEY}
```

Comment:
What is this typo btw

Reply:
It's yours :) Now that I think of it, I suppose this `docker-compose.ci.yml` should not need these variables exposed as this is only used by github actions and the tests shouldn't rely on having these variables anyway.

Reply:
Maybe we can remove all these in later PR.